File: 백준/Bronze/2480. 주사위 세개/주사위 세개.java
import java.util.*;
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        
        int dice1 = Integer.parseInt(st.nextToken());
        int dice2 = Integer.parseInt(st.nextToken());
        int dice3 = Integer.parseInt(st.nextToken());
        int top=0;

        if(dice1 == dice2 && dice2 == dice3) {
            System.out.print(10000+dice1*1000);
        }

        else if (dice1 == dice2 && dice1 != dice3) {
            System.out.print(1000+dice1*100);
        }
        else if (dice2==dice3 && dice2 != dice1) {
            System.out.print(1000+dice2*100);
        }
        else if (dice1==dice3 && dice2 != dice1) {
            System.out.print(1000+dice1*100);
        }
        else {
            top = Math.max(Math.max(dice1, dice2), dice3);

            System.out.print(top*100);
        }

    }
}

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.kt
import java.io.*
import java.util.*

fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val st = StringTokenizer(br.readLine())

    var d1 = Integer.parseInt(st.nextToken())
    var d2 = Integer.parseInt(st.nextToken())
    var d3 = Integer.parseInt(st.nextToken())

    println(getReward(d1, d2, d3))
}

fun getReward(d1: Int, d2: Int, d3: Int): Int {
    if (d1 == d2 && d2 == d3) return d1 * 1000 + 10000
    else if (d1 == d2 || d1==d3) return d1 * 100 + 1000
    else if (d2 == d3) return d2 * 100 + 1000
    else return d1.coerceAtLeast(d2.coerceAtLeast(d3)) * 100
}

